src/contact-summary/contact-summary-emitter.js

```diff
@@ -35,7 +45,7 @@ function emitter(contactSummary, contact, reports) {
         }
       }
     } else {
```

Comment:
this `else` block is maybe a proble:

if `appliesToType: ['person']` then we one run block
if `appliesToType: ['report']` then we run the other

what if `appliesToType: ['person', 'report']?` is this valid? how should we handle it?

In other parts of the configuration the `appliesToType` attribute is a filter on a `report.form` value - but here it behaves differently and filters by value 'report' only. we should clarify the documents for that - could you do another PR? or make an issue?

in the task/target code, we use the `appliesTo` attribute to select between contact vs report. And `appliesToType` to filter specific by contactType/form. So this code uses the same variable names but handles things quite differently. Strange!

Reply:
I think the same applies here, I don't see TL using `appliesToType: ['person', 'report']` .. I can't imagine cases where you want the same fields to be displayed for both a report card and a contact card. So in theory possible but, realistically, it will never happen.  

As for your second comment about `report.form`, I think the doc is already clear (https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md): in one case, `appliesToType` can be either form codes or contact types, in the other case, it is either `report` or `contact`. Let me know if I misunderstood your comment.

Reply:
> I can't imagine cases where you want the same fields to be displayed for both a report card and a contact card

Should we throw if 'report' is used together with other fields? Rather than having it just 'not work'? Then it is clear this is not supported.

Reply:
> I think the doc is already clear

This is the documentation for "appliesToType" in the link you sent.

> Filters the contacts for which appliesIf will be evaluated. For example, ['person'] or ['clinic', 'health_center'].

It doesn't even mention that you can use this value with the value 'report'. I'm suggesting that we clarify that.

Reply:
Sounds good. Let's throw .. I will add a test. 
Will update the doc.